### DISCUSSION TITLE
Welcome to NDui Discussions!

### DISCUSSION THREAD

**Discussion by siweia** [siweia/NDui#125] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xibaxd** [comment]
Hi, I have 2 questions about the addon.

1: The raid icon on the top that shows me how many players I have in my party, and u can click ready check, countdown vanished. How can I fix that?

2: Whisper chat is always on the default channel, even if I change on interface to pop up on a new tab, It will come back to default at some point. How can I fix that?

**Reply by siweia** [reply]
1. NDui Console, enable your raid tool. If enabled already, then you need to figure out which addon causes the conflicts.
2. It is forced to use default channel on each time login into game. I don't see any points to use other whisper mode. Only thing you can do is to delete the line at here. https://github.com/siweia/NDui/blob/master/Interface/AddOns/NDui/Modules/Chat/Core.lua#L342

**Comment by ShenloWoW** [comment]
HI,
Thank you for this ui, i recently turning back on WoW and i found all i need on this pack. (Sorry for my language, i'm french with a poor english).

I just have a question because i can't find the option and i would to know if that exist or not. 
When i change  my specialization Dps/ heal, for exemple, i have to change manually of profile. The option to attribuate a profile to a spec is available? I don't have found here atm.

Thx again

**Reply by siweia** [reply]
No, there isn't an option to switch automaticly. You have to switch profile manually.

**Reply by ShenloWoW** [reply]
Ok thanks, i'm looking for a long time because i'm this kind of people who don't see an elephant in front of me. <3

**Comment by uzi1778** [comment]
您好， 为什么没有2.4.3版本的
